pkg/reconciler/kubernetes/tektoninstallerset/client/custom_versioned_clustertask.go

```diff
@@ -28,14 +28,11 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-var (
-	versionedClusterTaskInstallerSet = "VersionedClusterTask"
-)
-
-// VersionedClusterTaskSet this is an exception case where we create one installer set for one minor version
-// not for patch version and we don't remove older installer sets on upgrade, hence keeping it different
+// VersionedTaskSet this is an exception case where we create one installer set for one minor version
+// not for patch version, and we don't remove older installer sets on upgrade, hence keeping it different
 // from custom set otherwise code becomes unnecessarily complex to handle this case
-func (i *InstallerSetClient) VersionedClusterTaskSet(ctx context.Context, comp v1alpha1.TektonComponent, manifest *mf.Manifest, filterAndTransform FilterAndTransform) error {
+func (i *InstallerSetClient) VersionedTaskSet(ctx context.Context, comp v1alpha1.TektonComponent, manifest *mf.Manifest,
+	filterAndTransform FilterAndTransform, isType, isName string) error {
```

Comment:
nit: generally I do not prefer to use `is` as prefix for non `bool` variables
May be,
`isType` => `insType`
`isName` => `insName`

Reply:
Updated